doc/modules/ensemble.rst

```diff
@@ -1095,6 +1095,44 @@ supported for multiclass context.
 
   * :ref:`sphx_glr_auto_examples_ensemble_plot_monotonic_constraints.py`
 
+.. _categorical_support_gbdt:
+
+Categorical Support
+-------------------
+
+For datasets with categorical data, :class:`HistGradientBoostingClassifier`
+and :class:`HistGradientBoostingRegressor` has native support for splitting
+on categorical features. This is often better than one hot encoding because
+it leads to faster training times and trees with less depth. When splitting
+a node, the categorical feature will be split into two subsets: one going to
+the left child and the other going to the right child. First, the histogram
+for each categorical feature is first sorted according to the ratio:
+`sum of gradient / sum of hessian` in each bin. Then the best split is found
+by considering splits along the stored histogram.
+
+If the cardinality of a categorical feature is greater than `max_bins`, then
+the top `max_bins` categories based on cardinality will be kept and the less
+frequent categories will be considered missing. If there are missing values
+during training, the missing will be considered its own category. When
+predicting, categories that were unknown during fit time, will be consider
+missing.
```

Comment:
I would just reorder the sentences:

> If there are missing values
during training, the missing values will be considered as a single category. When
predicting, categories that were unknown during fit time, will be consider
missing. If the cardinality of a categorical feature is greater than `max_bins`, then the top `max_bins` categories based on cardinality will be kept, and the less
frequent categories will be considered as missing.